test/test_c10d.py

```diff
@@ -1509,6 +1509,23 @@ def test_barrier_implies_wait(self):
         for i, tensor in enumerate(tensors):
             self.assertEqual(torch.full(size, float(i * self.world_size)), tensor)
 
+    def test_round_robin(self):
+        num_process_groups = 2
+        store = c10d.FileStore(self.file_name, self.world_size)
+        pg = c10d._round_robin_process_groups([
+            c10d.ProcessGroupGloo(
+                c10d.PrefixStore(str(i), store),
```

Comment:
will it be good to add comments somewhere to suggest users to use PrefixStore in sub process group?

Reply:
Good point.

This should be part of the general docs about the lower level c10d API. It's somewhat implied, since a single store being used by multiple process groups intuitively gets clobbered, but agree it would be good to include in docs somewhere.